src/query/__tests__/RelayQueryRoot-test.js

```diff
@@ -342,6 +342,77 @@ describe('RelayQueryRoot', () => {
     expect(meIdentifyingArg).toBeUndefined();
   });
 
+  it('returns the identifying number argument ', () => {
+    var nQuery = getNode(Relay.QL`
+      query {
+        task(number:5){
+          dumb,
+        }
+      }
+    `);
+    nQuery.getConcreteQueryNode().metadata= {
+      identifyingArgName: 'number',
+      identifyingArgType: 'scalar' };
```

Comment:
What happens if you don't assign `metadata` here? Will the test fail? If so, this means that this PR won't work in practice - we'll need to ensure that the metadata is set by the plugin.